pkg/pow/solutionHandler.go

```diff
@@ -225,8 +225,9 @@ func (s *SolutionHandler) submit(ctx context.Context, contract tellorCommon.Cont
 
 	txn, err := contract.SubmitSolution(
 		s.currentNonce,
-		s.currentChallenge.RequestID,
-		s.currentValue)
+		[5]*big.Int{s.currentChallenge.RequestID}, // ?????????????
+		[5]*big.Int{s.currentValue},               // ???????????????
```

Comment:
It seems that the interface for the `SubmitSolution` has changed from a single value input into an array of 5 values. Is that correct and how do you think we should handle it on the golang side?

Reply:
yeah, so it's just the array of 5, the single value was v1